src/types/index.ts

```diff
@@ -171,23 +172,21 @@ export interface SubscriptionIOS extends ProductCommon {
 
 export type Subscription = SubscriptionAndroid & SubscriptionIOS;
 
-export interface RequestPurchaseBaseAndroid {
+export interface RequestPurchaseAndroid {
+  skus?: Sku[];
```

Comment:
`Sku[]` is only for one-time purchases. Subscriptions require `SubscriptionOffer[]` instead

Reply:
Thanks for spotting that, I will revert it